### PR TITLE
[MI-170] Add description of Traits to readme

### PR DESCRIPTION
**Pull request by joseconsador**
Add description of Traits  usage to readme

/cc @miogalang @jwswj @mmolina @atroche 

### References
 - Jira link: https://zendesk.atlassian.net/browse/MI-170

### Risks
 - None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,6 +87,33 @@ The test suite is run via phpunit. Note that these are all live tests that must
 $tickets = $this->client->tickets()->sideload(array('users', 'groups'))->findAll();
 ```
 
+### Traits
+
+#### Resource Traits
+
+When adding a resource, use traits to define available API calls. As a standard, traits are added after class constants and arranged alphabetically when declared.
```
Comment (miogalang) [inline README.md]: Please mention that they should be grouped by adding whitespaces in between. Instantiator goes first, then single resource traits then last is the bulk traits

Comment (joseconsador) [inline README.md]: :haircut:

**Comment by miogalang** [comment]
:+1:

**Comment by miogalang** [comment]
I didnt notice you branched this off traits. This can go directly onto v2

**Comment by joseconsador** [comment]
closing merge to v2 instead

**Pull request closed without merging**